Promact.Oauth.Server/src/Promact.Oauth.Server/Repository/UserRepository/UserRepository.cs

```diff
@@ -227,18 +228,9 @@ public IEnumerable<UserAc> GetAllUsers()
         /// <returns>List of all Employees</returns>
         public async Task<List<UserAc>> GetAllEmployees()
         {
-            var employees = await _userManager.GetUsersInRoleAsync(_stringConstant.RoleEmployee);
-            var userList = new List<UserAc>();
-
-            foreach (var user in employees)
-            {
-
-                var listItem = _mapperContext.Map<ApplicationUser, UserAc>(user);
-
-                userList.Add(listItem);
-            }
-
-            return userList.OrderBy(users => users.FirstName).ToList();
+            var users =await _userManager.Users.Where(user=>user.IsActive).ToListAsync() ;
```

Comment:
Why can't you do an OrderBy directly after Where clause? this way method would just be of 2 lines.